doc/interface.rst

```diff
@@ -426,6 +426,12 @@ Tool Bar
   The associated pop-up
   allows the color of the displayed residuals to be changed.
 
+:icon:`location` Edit Camera Registration Points
```

Comment:
```suggestion
:icon:`location` Edit Registration Points
```
These *should* all match the `text` of the actions. (Yes, I know, what that text *is* is not obvious. I admit I have to look at the `.ui`...)

Reply:
If the use can't see that text in the application then why does it matter?

Should the action be changed to "Edit Camera Registration Points" instead?  Everywhere else we are talking about Camera Registration Points.  If we drop the "Camera" it's less clear that we are talking about the same thing.

Reply:
> If the user can't see that text in the application then why does it matter?

We might turn on text+icon, in which case it would be unfortunate if these are not consistent. (I'm not sure we have anything that allows "follow style"...)

> Should the action be changed to "Edit Camera Registration Points" instead?

That's an option, although it would be *quite* long if that ever actually appeared in the toolbar.